product-service/src/main/java/com/minhaz/orders101/endpoints/ProductsEndpoint.java

```diff
@@ -103,14 +108,54 @@ private Object notFoundError(String id) {
     })).build();
   }
 
+  @PATCH
```

Comment:
i'm not expecting to see these changes in this MR - just the tests.

Is this because a lot of these changes aren't in main?